.ci/packaging.groovy

```diff
@@ -263,9 +260,8 @@ def publishArtifactsDev() {
   uploadArtifacts(bucketLocation: "gs://${JOB_GCS_BUCKET}/${URI_SUFFIX}")
 
   dockerLogin(secret: env.DOCKER_SECRET, registry: env.DOCKER_REGISTRY)
-  dir("${BASE_DIR}"){
-    sh(label: 'Push', script: "./.ci/scripts/push-docker.sh ${env.GIT_BASE_COMMIT} ${env.DOCKER_IMAGE}")
-  }
+  sh(label: 'Tag Docker image', script: "docker tag ${dockerImage} ${env.DOCKER_IMAGE}:${env.GIT_BASE_COMMIT}")
+  sh(label: 'Push Docker image', script: "docker push ${env.DOCKER_IMAGE}:${env.GIT_BASE_COMMIT}")
```

Comment:
Nit: We often wrap this in a retry like `retryWithSleep(retries: 3, seconds: 5, backoff: true){`

Reply:
Thanks! Added.